pallets/nfts/src/tests.rs

```diff
@@ -2466,138 +2513,45 @@ fn approvals_limit_works() {
 #[test]
 fn approve_collection_transfer_works() {
 	new_test_ext().execute_with(|| {
-		let (collection, locked_collection) = (0, 1);
-		let item = 42;
+		let (collection_id, locked_collection_id) = (0, 1);
 		let collection_owner = account(1);
-		let item_owner = account(2);
 		let delegate = account(3);
+		let item_id = 42;
+		let item_owner = account(2);
+
+		for origin in [root(), none()] {
```

Comment:
Can we add some comments here to explain these different origin tests.

Would also be good to clarify that this test is for `approve_collection_..` and `force_approve_collection_..`. Same for the other two cases (cancel_collection_approval & clear_collection_approvals)